src/components/Review/ReviewSection.tsx

```diff
@@ -1,23 +1,46 @@
 import React, { useState } from 'react'
```

Comment:
General comment: under each Question, we should see the result of the decision (Approve/Re-submit) along with the comment, right? Is there another issue to do this, or should it be part of this issue?

Reply:
Same problem as explained before.